src/components/molecules/event/eventBody/EventBody.tsx

```diff
@@ -157,10 +181,12 @@ export const EditEvent: React.FC<{ event: Event; setEdit: () => void }> = ({
               error={fields['description'].error}
               style={{ boxSizing: 'border-box', width: '100%' }}
             />
-            <div className={styles.imgContainer}>
-              <label>Endre arrangement bilde </label>
-              <input type="file" accept="image/*" onChange={handleFileUpload} />
-            </div>
+            <FileSelector
+              setFile={setFile}
+              text="Endre arrangement bilde"
+              accept="image/png"
```

Comment:
Should we only allow png? What about jpg, jpeg etc?

Reply:
As of now, the API only support PNG so this can be removed when multiple files types are supported